### ISSUE TITLE
Glasses object anchors?

### ISSUE THREAD

**Issue by jilleJr** [OpenMods/OpenPeripheral-Addons#69]
I've been searching like a madman but I can't find how to change the anchors of an object?

I was reading the changelog of OpenPeripherals Addon from 0.2.0 to 0.3.1 [here (link)](https://feedthebeast.atlassian.net/wiki/display/PML/Direwolf20+%281.7.x%29+1.3.0) and this is one important change I saw:

```- anchors (allow positions elements in screen corners and middle)```

I am currently using Direwolf20 1.7.10 pack v1.4.1 which includes OpenPeripherals Addon v0.3.1 and I can not find a method anywhere that would change any anchors.
I have been looking around each object type but with no luck.

I even looked at [the source code (link)](https://github.com/OpenMods/OpenPeripheral-Addons/blob/master/src/main/java/openperipheral/addons/glasses/drawable/Drawable.java) to see if there is something I missed, and I found methods called `setScreenAnchor`, `setObjectAnchor`, and `setAlignment`. Now I am not experienced with java so excuse me if I'm wrong but shouldn't I be able to access them from within computercraft?

![image](https://cloud.githubusercontent.com/assets/2477952/8289840/2df19d92-191f-11e5-83e2-1c5301af3425.png)

Maybe I'm totally wrong with this? In the changelog it specifies that it's possible to modify anchors for elements on the screen, I just presumed that meant the glasses but maybe it's for something else? Like some API for java addons? Or maybe it's for the item selector peripheral?

----

### Tl;dr:
I want to change the anchors for my *terminal glasses* objects but I can't find the methods. Help! :scream:

**Comment by boq** [comment]
Yeaaaah, sorry for that. Known problem.

You can either downgrade to 0.3, use [this](http://www.openmods.info:8080/job/OpenPeripheralAddons/197/artifact/build/libs/OpenPeripheralAddons-1.7.10-0.3.1-snapshot-197.jar) or wait until I release (end of this week maybe? I'm terrible at estimations...).

**Comment by jilleJr** [comment]
Dangit, was really hoping to start using this. Oh well.

I was watching [this script (link)](https://gist.github.com/boq/67300900797b113a124d) and got confused that maybe they were using a newer non-released version...

Is openperipherals serverside only? Cause I'm playing on a server and I can ask the owner to update/downgrade a mod but if it requires everyone to do it then no-go

**Comment by boq** [comment]
Well, theoretically OpenPeripheral-Addons is needed on both client and server, but [this file](http://www.openmods.info:8080/job/OpenPeripheralAddons/197/artifact/build/libs/OpenPeripheralAddons-1.7.10-0.3.1-snapshot-197.jar) is 100% compatible with released version, and just needs to be replaced on server.

**Comment by jilleJr** [comment]
Cool thank yooo
